sites/platform/static/files/fetch/examples/relationships/influxdb

```diff
@@ -2,8 +2,11 @@
     "service": "influxdb18",
     "ip": "169.254.244.110",
     "hostname": "duqbjfn7t4dwr2fi2o7bsvqafy.influxdb18.service._.eu-3.platformsh.site",
-    "cluster": "rjify4yjcwxaa-master-7rqtwti",
     "host": "influxdb.internal",
+    "cluster": "rjify4yjcwxaa-master-7rqtwti",
```

Comment:
please do not expose potential IPs ;) 

for such config, please use: 
- IP: 123.456.78.90
-  "hostname": "azertyuiopqsdfghjklm.influxdb18.service._.eu-1.platformsh.site",